cmd/wksctl/profile/enable/enable.go

```diff
@@ -87,3 +108,26 @@ func profileEnableRun(params profileEnableFlags) error {
 
 	return nil
 }
+
+func processIgnoreFile(dir string, reader io.Reader) ([]string, error) {
+	result := []string{}
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
+		pathToIgnore := strings.Trim(scanner.Text(), " ")
```

Comment:
This will trick users who put a comment as a line suffix. For example:

    ignored_file.yaml # ignored because something

Suggestion: use a regex like `^\s*([^\s#]+).*$` to match a line in a more robust way. https://regex101.com/r/TLrtUU/1

Reply:
fixed per your suggestion!